languages/common-lisp/exercises/concept/lists/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+Leslie the Lisp Alien needs to do some shopping. It is very important to have a shopping list. One needs to add things to it, remove things from it, and even look if something is on the list for instance.
```

Comment:
I like the shortened version!

Reply:
Thank you for this comment - not for the encouragement and positive feedback - but for putting it here where I can see I failed to edit something!!!!